vector/src/main/java/im/vector/app/core/extensions/ViewExtensions.kt

```diff
@@ -58,3 +59,14 @@ fun ImageView.setDrawableOrHide(drawableRes: Drawable?) {
     setImageDrawable(drawableRes)
     isVisible = drawableRes != null
 }
+
+fun TextView.setText(text: String?, nullVisibility: Int, vararg relatedViews: View) {
```

Comment:
I do not know why my comment is not referring to this line, nor why I cannot edit the comment. The nit is about this extension.